### ISSUE TITLE
Clarify `apm-server.ssl.certificate_authorities` 

### ISSUE THREAD

**Issue by simitt** [elastic/apm-server#3198]
Explicitly mention that `apm-server.ssl.certificate_authorities` should only be set when the agent is sending a client certificate, and cross reference to [client_authentication](https://www.elastic.co/guide/en/apm/server/current/agent-server-ssl.html#_client_authentication) being required, if it is set.

Also add a sentence for clarification in the [apm-server.yml](https://github.com/elastic/apm-server/blob/master/apm-server.yml#L97)

**Comment by karmi** [comment]
I'd add something like this to the `apm-server.yml`:

```
# Configure a list of root certificate authorities for verifying client certificates.
# Enable ONLY when you want to use and verify client certificates.
#certificate_authorities: []
```

**Comment by insukcho** [comment]
> If certificate_authorities has been specified, this setting will automatically change to required.

@bmorelli25 This comment is from https://www.elastic.co/guide/en/apm/server/current/agent-server-ssl.html#_client_authentication.

Could you please add this in [apm-server.yml](https://github.com/elastic/apm-server/blob/master/apm-server.yml#L125) something like below? 

```
    # Configure which type of client authentication is supported.
    # Options are `none`, `optional`, and `required`. Default is `optional`.
    # If certificate_authorities has been specified, this setting will 
    # automatically change to `required`.
    #client_authentication: "optional"
```

**Comment by simitt** [comment]
closed with #3500
